src/Frontend/src/composables/serviceLicense.ts

```diff
@@ -13,32 +13,32 @@ interface License extends LicenseInfo {
   formattedInstanceName: ComputedRef<string>;
 }
 
-const emptyLicense: License = {
-  edition: "",
-  expiration_date: "",
-  upgrade_protection_expiration: "",
-  license_type: "",
-  instance_name: "",
-  trial_license: true,
-  registered_to: "",
-  status: "",
-  license_status: LicenseStatus.Unavailable,
-  license_extension_url: "",
-  licenseEdition: computed(() => {
-    return `${license.license_type}${license.edition ? `, ${license.edition}` : ""}`;
-  }),
-  formattedInstanceName: computed(() => {
-    return license.instance_name || "Upgrade ServiceControl to v3.4.0+ to see more information about this license";
-  }),
-  formattedExpirationDate: computed(() => {
-    return license.expiration_date ? new Date(license.expiration_date.replace("Z", "")).toLocaleDateString() : "";
-  }),
-  formattedUpgradeProtectionExpiration: computed(() => {
-    return license.upgrade_protection_expiration ? new Date(license.upgrade_protection_expiration.replace("Z", "")).toLocaleDateString() : "";
-  }),
-};
-
-const license = reactive<License>(emptyLicense);
+class emptyLicense implements License {
```

Comment:
I have decided to refactor this type to be a class so we can reference its data directly.
The previous implementation was not giving the correct result.